include/Metabox/Posts/CustomTaxonomy.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+defined( 'ABSPATH' ) || exit; // Exit if accessed directly.
+
+/**
+ * Metabox to move posts based on category Class.
+ *
+ * @since 2.0.0
+ */
+class BM_Metabox_Posts_CustomTaxonomy extends BM_Metabox_Base {
+
+	protected function initialize() {
+		$this->meta_box_slug = 'bm-posts-by-custom-taxonomy';
+		$this->messages['box_label'] = __( 'Move Posts By Custom Taxonomy', 'bulk-move' );
+		$this->action = 'move_custom_taxonomy';
+	}
+
+	public function render() {
+		?>
+
+		<!-- Custom Taxonomy Start-->
+
+		<fieldset class="options">
+			<table class="optiontable">
+				<tr>
+					<td scope="row" colspan="2">
+						<?php _e( 'Select the post type to show its custom taxonomy.', 'bulk-move' ); ?>
+					</td>
+					<td scope="row">
+				</tr>
+
+				<tr>
+					<td scope="row" colspan="2">
+						<select name="smbm_mbct_post_type" id="smbm_mbct_post_type">
```

Comment:
Abstract this into a helper function that is added to the base class

Reply:
> Abstract this into a helper function that is added to the base class

Fixed